sklearn/svm/classes.py

```diff
@@ -491,6 +491,22 @@ class SVC(BaseSVC):
     intercept_ : array, shape = [n_class * (n_class-1) / 2]
         Constants in decision function.
 
+    fit_status_ : int
+        0 if correctly fitted, 1 otherwise (will raise warning)
+
+    probA_, probB_ : array, shape = [n_class * (n_class-1) / 2]
```

Comment:
Each should be on its own line. Though you only need the explanation once.
Otherwise it looks greatk.